tests/unit/integrations/bitbucket/test_bitbucket.py

```diff
@@ -871,3 +871,238 @@ def test_initialize_repository_for_runtime_without_bitbucket_token(
     assert ProviderType.GITHUB in provider_tokens
     assert ProviderType.GITLAB in provider_tokens
     assert ProviderType.BITBUCKET not in provider_tokens
+
```

Comment:
@openhands we seem to be using direct comparisons for called args instead of using `assert_called_with()` 

please rework the tests in this file so that we rely on `assert_called_with()`